src/wallet/wallet2.h

```diff
@@ -630,6 +630,7 @@ namespace tools
     std::string get_subaddress_label(const cryptonote::subaddress_index& index) const;
     void set_subaddress_label(const cryptonote::subaddress_index &index, const std::string &label);
     void set_subaddress_lookahead(size_t major, size_t minor);
+    std::pair<size_t, size_t> get_subaddress_lookahead() const { return {m_subaddress_lookahead_major, m_subaddress_lookahead_minor}; }
```

Comment:
uint32_t

Reply:
I wrote this way because m_subaddress_lookahead_major is of size_t (which should’ve been uint32_t). I’m not sure if the type difference could make a real difference.